airflow/jobs.py

```diff
@@ -602,6 +602,9 @@ def manage_slas(self, dag, session=None):
                             dag_id=ti.dag_id,
                             execution_date=dttm,
                             timestamp=ts))
+                        Stats.incr('task_sla_miss', 1, 1, tags=['dag_id:{}'.format(ti.dag_id)])
```

Comment:
A simple convenience method can help DRY things up.
Something along the lines of `task_stat_incr(metric_name, task_instance)` would cover a lot of these cases.